app/lib/common/notifications/notifications.dart

```diff
@@ -312,86 +312,191 @@ Future<bool> handleMessage(
   final deviceId = message.data!['device_id'] as String;
   final roomId = message.data!['room_id'] as String;
   final eventId = message.data!['event_id'] as String;
+  debugPrint('Received msg $roomId: $eventId');
   final payload =
       makeForward(roomId: roomId, deviceId: deviceId, eventId: eventId);
```

Comment:
Old fallback I didn't touch. It was a common "we don't know what it is but can inform the app about the details so it can figure it out"-fallback link ... Lemme check the code if we still want any of that...

Reply:
gone.